packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -109,112 +108,103 @@ export default function GridItem( { categoryId, composite, icon, item } ) {
 	}
 
 	return (
-		<>
-			<div className={ patternClassNames }>
-				<CompositeItem
-					className={ previewClassNames }
-					role="option"
-					as="div"
-					{ ...composite }
-					onClick={ item.type !== PATTERNS ? onClick : undefined }
-					onKeyDown={ isUserPattern ? onKeyDown : undefined }
-					aria-label={ item.title }
-					aria-describedby={
-						ariaDescriptions.length
-							? ariaDescriptions
-									.map(
-										( _, index ) =>
-											`${ descriptionId }-${ index }`
-									)
-									.join( ' ' )
-							: undefined
-					}
+		<li className={ patternClassNames }>
+			<button
+				className={ previewClassNames }
+				onClick={ item.type !== PATTERNS ? onClick : undefined }
+				onKeyDown={ isUserPattern ? onKeyDown : undefined }
```

Comment:
Now that the Actions button is keyboard accessible, I'd remove the onKeyDown to delete a pattern via keyboard delete / backspace.